src/pyfmi/fmi_algorithm_drivers.py

```diff
@@ -102,11 +99,9 @@ class AssimuloFMIAlgOptions(OptionBase):
             Default: "Default"
 
         dynamic_diagnostics --
-            If True, enables logging of diagnostics data to the binary result file. This requires that
-            the option 'result_handler' is an instance of ResultHandlerBinaryFile, otherwise an exception is raised.
-            Model variable names are not allowed to start with 'Diagnostics' using this option
-            and a check for this is performed before simulation start. If this criteria is not
-            met an exception is raised.
+            If True, enables logging of diagnostics data to a result file. This requires that
+            the option 'result_handler' supports 'dynamic_diagnostics', otherwise an exception is raised.
```

Comment:
Can be good to mention what kind of exception is raised

Reply:
Fixed.